frontend/src/style/global.css

```diff
@@ -143,8 +143,8 @@ p {
   color: var(--button_dark);
 }
 
-/* effect on desktop*/
-@media (min-width: 600px) {
+/* effect on desktop */
+@media only screen and (min-width: 1024px) {
```

Comment:
you can take references (effect on desktop / ipad / phone) when dealing with the interface on different devices

Reply:
move to https://github.com/jykuo-love-shiritori/twp/pull/68#discussion_r1408927584.